tests/Aspire.Hosting.Tests/Snapshots/ProjectResourceTests.ProjectResourceWithContainerFilesDestinationAnnotationWorks.verified.txt

```diff
@@ -0,0 +1,7 @@
+﻿ARG source_IMAGENAME=myimage:latest
```

Comment:
Nit: We probably want these to be all upper case to match the traditional format in Dockerfiles.

Reply:
cc @vhvb1989